csrc/preseg_passes/allocation_order_inference.cpp

```diff
@@ -15,30 +15,66 @@ namespace nvfuser::preseg_passes {
 
 namespace {
 
+// returns allocation order to propagate adjusted for `tv`
+// Right now it merely removes all the entry corresponding to reduction iter
+// domains in tv,
+AllocationOrder adjustAllocationOrder(
+    const TensorView* tv,
+    const AllocationOrder& alloc_order) {
```

Comment:
I'm no longer sure I follow the code. Is `alloc_order` the allocation order of `tv` or that of its producer?

Reply:
alloc_order is the allocation order of `tv`. I see this as a failure in the comment. :bow: